packages/core/adder/options.ts

```diff
@@ -57,6 +58,14 @@ export type AvailableCliOption = {
 export type AvailableCliOptions = Record<AvailableCliOptionKeys, AvailableCliOption>;
 
 export const availableCliOptions: AvailableCliOptions = {
+    default: {
+        cliArg: "default",
+        processedCliArg: "default",
+        type: "boolean",
+        default: false,
+        description: "Installs default adder options",
```

Comment:
Should we add something to the description, indicating that `--default` can be used in combination with other adder options, like you did in the PR description?

Reply:
We absolutely could! Would `Installs default adder options for unspecified options` suffice, or would that be _too_ vague?

Reply:
I think that's better than what we have right now and I don't have any better wording to suggest. I have just update the text.